src/squeezed.ml

```diff
@@ -29,17 +29,33 @@ let options = [
 	"domain-zero-dynamic-max", Arg.String (fun x -> Squeeze.domain_zero_dynamic_max := if x = "auto" then None else Some (Int64.of_string x)), (fun () -> match !Squeeze.domain_zero_dynamic_max with None -> "using the static-max value" | Some x -> Int64.to_string x), "Maximum memory to allow domain 0";
 ]
 
+module S=Memory_interface.API(Idl.GenServerExn ())
+
+let bind () =
```

Comment:
Being this the first PR using the new ppx mechanism, I think it would be worth to document line 32 and the content of 34. Just to make it more explicit that we are 'binding' the declared calls in the idl with the implementations provided in `memory_server`.